navigation.yaml

```diff
@@ -3,27 +3,18 @@ openstack:
   path: /openstack
 
   children:
-    - title: Overview
-      path: /openstack
```

Comment:
I think we need to keep the Overview child, otherwise the secondary nav doesn't get rendered in /openstack